test/lan_udev_auto_test.rb

```diff
@@ -4,9 +4,11 @@
 
 require "yast"
 
-module Yast
+include Yast::UIShortcuts
```

Comment:
not needed if you use `require "yast/rspec"` helpers already include UIShortcuts and add it only in rspec scope. See https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/rspec/shortcuts.rb